src/scripts/generate-schemas.ts

```diff
@@ -190,16 +190,20 @@ function joinStr(xs: string[]): string {
     return xs.map(quote).join(" | ");
 }
 
-const instances = {
-    baseUrl: "https://admin:district@play.dhis2.org/",
-    versions: ["2.30", "2.31", "2.32", "2.33"],
-};
+type Instance = { version: string; url: string };
 
-async function generateSchema(version: string) {
-    const url = instances.baseUrl + version;
+const instances: Instance[] = [
+    { version: "2.30", url: "http://admin:district@localhost:8030" },
```

Comment:
We should probably freeze the schemas for 30 and 31 by removing them from this array but keeping the last generated type definition file. The refresh is meant to detect changes pushed to newer revisions but that won't happen on these versions as are off support

Reply:
I thought about it, and it makes sense when just pulling upstream schema changes. However, what we often do is change our own schemas, and we'd want these new feature also for the old versions. 

What we can do is: add an option to generate-schemas to specify which versions will be updated, or if we want to include the deprecated ones.

Reply:
Yeah, mostly my comment was because I don't really want it to assume those instances will be up on a given port for different versions (even though a 80XX convention as-is makes sense to me). Something in between that allows to pass a subset of deprecated versions that you want to additionally include works for me.